### PR TITLE
Attach the onSubmit handler to the button

### PR DESCRIPTION
**Pull request by itsjoekent**
### What does this PR do?
Fixes the upload button!

In [this](https://github.com/DoSomething/phoenix-next/pull/571/files#diff-201edebec1634a925cb7e8371eba4809R137) PR the form became a sibling of the submit button, so the form on submit no longer has anything to trigger it. This PR attaches the submit to the form.

### CODE REVIEW

File: resources/assets/components/ReportbackUploader/ReportbackUploader.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ class ReportbackUploader extends React.Component {
     );
 
     const reportbackUploader = (
-      <form className="reportback-form" onSubmit={this.handleOnSubmitForm} ref={form => (this.form = form)}>
```
Comment (mendelB) [inline resources/assets/components/ReportbackUploader/ReportbackUploader.js]: I don't know if this really matters, but do we care about removing the ability to submit the form with the 'Return'/'Enter' button?

Comment (itsjoekent) [reply resources/assets/components/ReportbackUploader/ReportbackUploader.js]: mmm, good point we could still leave the onSubmit handler here

Comment (weerd) [reply resources/assets/components/ReportbackUploader/ReportbackUploader.js]: Hmm, that's a good question...

**Comment by itsjoekent** [comment]
@mendelB so the enter button works on this PR now, but you have to hit enter on anything besides the text area (Like you mentioned in Slack)

**Pull request merged**